src/displayapp/screens/Dice.cpp

```diff
@@ -0,0 +1,231 @@
+#include "Dice.h"
+
+#include "Screen.h"
+#include "Symbols.h"
+#include "lvgl/lvgl.h"
+
+using namespace Pinetime::Applications::Screens;
+const uint8_t Dice::diceSizes[] = {2, 3, 4, 6, 8, 12, 20, 100};
```

Comment:
The "standard" dice set also includes d10.

Reply:
Oh, yep, you're absolutely right -- missed the d10 somehow.  I'll make sure that gets included with future changes.

Reply:
@jonvmey Fixed this in the latest changes.

Reply:
Couldn't the value be initialized in the header instead?

Reply:
@Riksu9000 Added this in https://github.com/JF002/InfiniTime/pull/565/commits/7c0174284033c67c6e11b61d2829570715e35cb0